docs/source/api/index.rst

```diff
@@ -24,6 +24,7 @@ JupyterHub API Reference:
 
 .. toctree::
 
+    app
```

Comment:
Causing a build failure in docs since `app.rst` does not exist in `docs/source/api` directory.